consultation_analyser/consultations/views/sessions.py

```diff
@@ -14,6 +14,7 @@
 
 def send_magic_link_if_email_exists(request: HttpRequest, email: str) -> None:
     try:
+        email = email.lower()
         user = User.objects.get(email=email)
```

Comment:
Do we also need to lower the email address before it is saved in the User table?